meta-xt-prod-devel-rcar-control/recipes-guests/doma/files/doma-vdevices-virtio.cfg

```diff
@@ -0,0 +1,32 @@
+# =====================================================================
+# DomA virtio virtual devices
+# =====================================================================
+
+virtio=[
+'backend=DomD, type=virtio,device, transport=pci, bdf=0000:00:01.0, grant_usage=1, backend_type=qemu',
+'backend=DomD, type=virtio,device, transport=pci, bdf=0000:00:02.0, grant_usage=1, backend_type=qemu',
+'backend=DomD, type=virtio,device, transport=pci, bdf=0000:00:03.0, grant_usage=1, backend_type=qemu',
+'backend=DomD, type=virtio,device, transport=pci, bdf=0000:00:04.0, grant_usage=1, backend_type=qemu',
+'backend=DomD, type=virtio,device, transport=pci, bdf=0000:00:05.0, grant_usage=1, backend_type=qemu'
+]
+
+device_model_args=[
+'-device', 'virtio-net-pci,disable-legacy=on,iommu_platform=on,bus=pcie.0,addr=1,romfile=,id=nic0,netdev=net0,mac=08:00:27:ff:cb:ce',
+'-netdev', 'type=tap,id=net0,ifname=vif-emu,br=xenbr0,script=no,downscript=no',
+'-device', 'virtio-keyboard-pci,disable-legacy=on,iommu_platform=on,bus=pcie.0,addr=2',
+'-audiodev', 'alsa,id=snd0,out.dev=default',
+'-device', 'virtio-snd-pci,audiodev=snd0,disable-legacy=on,iommu_platform=on,bus=pcie.0,addr=3',
+'-device', 'virtio-gpu-gl-pci,disable-legacy=on,iommu_platform=on,bus=pcie.0,addr=4',
+'-display', 'sdl,gl=on',
+'-vga', 'std',
+'-device', 'virtio-blk-pci,scsi=off,disable-legacy=on,iommu_platform=on,bus=pcie.0,addr=5,drive=image',
+'-drive', 'if=none,id=image,format=raw,file=/dev/STORAGE_PART3',
+'-d', 'guest_errors',
+'-monitor', 'telnet:127.0.0.1:1234,server,nowait',
```

Comment:
Just a reminder, for this to apply we will need some changes to Xen toolstack code to not apply a default value for this property internally.

Reply:
Hi @otyshchenko1, I've added that to my team's backlog. So, I guess, we will not forget about this topic.